### PR TITLE
[167756788] AdColony - Android - SDK causes the app to crash when trying...

### PR DESCRIPTION
**Pull request by hkosacki**
... to show an interstitial of AdColony when zone type is set to V4VC

Release of recent AdColony fixes (NPE and mixing up zone IDs).

### CODE REVIEW

File: android-adapters/changelog-adcolony.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
 #AdColony current versions:
 
-Adapter: 1.1.2
+Adapter: 1.1.
```
Comment (gabriel-fortin) [inline android-adapters/changelog-adcolony.md]: 3

**Comment by hkosacki** [comment]
typos fixed

File: android-adapters/changelog-adcolony.md (hunk 1 above)
Comment (hkosacki) [inline android-adapters/changelog-adcolony.md]: fixed

**Comment by gabriel-fortin** [comment]
review OK

**Comment by hkosacki** [comment]
PR rebased.

**Pull request merged**